src/core/include/databases.h

```diff
@@ -35,45 +36,66 @@ class Layer {
 };
 
 // class with net and temperatures
-// TODO: Fix it in future to make handy
 class HeatmapStorage {
  public:
-	using Heatmaps   = std::vector<Heatmap>;
 	HeatmapStorage() = default;
-	HeatmapStorage(std::vector<float> x_steps_mv, std::vector<float> y_steps_mv,
-	               const std::vector<float>& temperature, Box3D design_borders);
+	HeatmapStorage(std::vector<float> x_steps, std::vector<float> y_steps,
+	               const std::vector<float>& temperature,
+	               float environment_temperature, Box3D representation_borders);
 	[[nodiscard]] Box3D representation_borders() const {
 		return representation_borders_;
 	}
-	[[nodiscard]] float MinStep() const;
+	[[nodiscard]] float environment_temperature() const {
+		return env_temp_;
+	}
+
+	[[nodiscard]] float x_size() const {
+		return representation_borders_.coordinates()[0].second;
+	}
+	[[nodiscard]] float y_size() const {
+		return representation_borders_.coordinates()[1].second;
+	}
```

Comment:
How often do we use them? If just couple times then please delete it and don't polute code with such functions